src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java

```diff
@@ -2050,9 +2050,14 @@ public void configure(Transport transport) {
                     Map<String, List<String>> remoteBranches = new HashMap<String, List<String>>();
                     for (Ref remoteBranchRef : resultRemoteBranches) {
                         String branchFullName = remoteBranchRef.getName().replace(Constants.R_REMOTES, "");
-                        String[] tokens = branchFullName.split("/");
-                        String remotePart = tokens[0];
-                        String branchNamePart = tokens[1];
+                        String remotePart = StringUtils.EMPTY;
+                        String branchNamePart = StringUtils.EMPTY;
+                        int slashIndex = branchFullName.indexOf("/");
```

Comment:
Will this work on Windows?

Reply:
That is git naming convention for references (not a file system).